### ISSUE TITLE
Document missing title element

### ISSUE THREAD

**Issue by Sukha11y** [Sukha11y/Test-ZT-fix#7]
Problem
Where is it located
index.html - [https:///c%3A/Users/SukhadaPawar/Test-ZT-fix/Test-ZT-fix/index.html](https://c/Users/SukhadaPawar/Test-ZT-fix/Test-ZT-fix/index.html)

Locate element
XPath or CSS selector
html

Copy selector to clipboard
HTML element
<html style="">...</html>

Copy HTML element to clipboard
Description
A document title element is not located in a non-iframe DOM.
Why it matters
When there is no or an empty element, users may not know the specific purpose or topic of each page. The title of the page will be the URL instead which may not be meaningful.

Solution
What to do
When not in an iframe, ensure each page provides one non-empty title element within the head element that describes the topic or purpose of the page.
